dash/_dash_renderer.py

```diff
@@ -1,6 +1,6 @@
 import os
 
-__version__ = "1.19.0"
+__version__ = "1.19.1"
```

Comment:
This is interesting, I didn't realize we independently bump `dash_renderer`.

Reply:
It's automatically updated by the build from the package.json